### ISSUE TITLE
uhshax crashes on second boot

### ISSUE THREAD

**Issue by NWPlayer123** [hexkyz/hexFW#4]
trying to dump boot1, I get to the menu just fine the first time, reboots, go back into browser, run it again, and then it crashes before getting to "Waiting...."

**Comment by billy-acuna** [comment]
It works for me from CBHC.

**Comment by piratesephiroth** [comment]
yep, works fine here if I boot fw.img using haxchi (can't bother with that browser exploit)
